Spigot-Server-Patches/0667-Add-ElderGuardianAppearanceEvent.patch

```diff
@@ -0,0 +1,67 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
+Date: Mon, 1 Feb 2021 20:32:19 -0500
+Subject: [PATCH] Add ElderGuardianAppearanceEvent
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityGuardianElder.java b/src/main/java/net/minecraft/server/EntityGuardianElder.java
+index b691e844953bcc2853a806a3bbf9cb7338e98266..6c094da81b86bc491d28622907e5edaecd73de0a 100644
+--- a/src/main/java/net/minecraft/server/EntityGuardianElder.java
++++ b/src/main/java/net/minecraft/server/EntityGuardianElder.java
+@@ -58,16 +58,21 @@ public class EntityGuardianElder extends EntityGuardian {
+             boolean flag1 = true;
+             boolean flag2 = true;
+             boolean flag3 = true;
+-            Iterator iterator = list.iterator();
+ 
+-            while (iterator.hasNext()) {
+-                EntityPlayer entityplayer = (EntityPlayer) iterator.next();
```

Comment:
Please do not change the indentation of this code - simply wrap it with the new event check.